src/.vuepress/components/ExtensionsList.vue

```diff
@@ -5,19 +5,76 @@ The url argument must be of the form "https://paperback-ios.github.io/extensions
 -->
 
 <template>
-	<div>
-		<ul id="v-for-object">
-			<li v-for="extension in extensions"
-			:id="extension.name"
-			:key="extension.id"
-			>
-				{{ getName(extension.name) }}
-				<span v-for="tag in extension.tags" :key="tag.text"> <el-tag :type="tag.type" size="mini" effect="dark">{{ tag.text }}</el-tag> </span>
-			</li>
-		</ul>		
+	<div style="display: contents">
```

Comment:
Is there a reason to not put the `display: contents` in the `style` component?
I don't know if it would be better or not...